src/main/java/com/twilio/type/Twiml.java

```diff
@@ -0,0 +1,12 @@
+package com.twilio.type;
+
+public interface Twiml {
+
+    /**
+     * Returns the TwiML String.
+     *
+     * @return TwiML String
+     */
+    public String getTwiml();
```

Comment:
There's a piece missing here in that nothing implements `getTwiml()`. Using `Endpoint` as an example, it's an interface for multiple implementations: `Client`, `Sip`, and `PhoneNumber`.

Reply:
Doh! I Should just make it a class that takes a String in it's constructor...sorry about that!

Reply:
@childish-sambino This look better? Added to Promoter as well and it will use the Java `toString()` default for serialization .